worker/meson.build

```diff
@@ -277,6 +277,57 @@ executable(
   cpp_args: cpp_args + ['-DMS_EXECUTABLE'],
 )
 
+flatbuffers_schemas = [
+  'fbs/activeSpeakerObserver.fbs',
+  'fbs/audioLevelObserver.fbs',
+  'fbs/common.fbs',
+  'fbs/consumer.fbs',
+  'fbs/dataConsumer.fbs',
+  'fbs/dataProducer.fbs',
+  'fbs/directTransport.fbs',
+  'fbs/log.fbs',
+  'fbs/message.fbs',
+  'fbs/notification.fbs',
+  'fbs/pipeTransport.fbs',
+  'fbs/plainTransport.fbs',
+  'fbs/producer.fbs',
+  'fbs/request.fbs',
+  'fbs/response.fbs',
+  'fbs/router.fbs',
+  'fbs/rtpObserver.fbs',
+  'fbs/rtpParameters.fbs',
+  'fbs/rtpStream.fbs',
+  'fbs/rtxStream.fbs',
+  'fbs/sctpAssociation.fbs',
+  'fbs/sctpParameters.fbs',
+  'fbs/srtpParameters.fbs',
+  'fbs/transport.fbs',
+  'fbs/webRtcServer.fbs',
+  'fbs/webRtcTransport.fbs',
+  'fbs/worker.fbs',
+]
+
+flatbuffers_cpp_out_dir = 'FBS'
+flatbuffers_cpp_install_dir = meson.current_source_dir() + '/include'
+
+flatc = find_program('flatc')
```

Comment:
Where this this find_program() look for flatc binary? In the PATH? Because it shouldn't.

Reply:
It's not in the path. flatc is a subproject of mediasoup. It finds it there as the subproject installation is a dependency for the target.

Reply:
What I would like to hear is that it just uses the binary provided by the flat subproject. If it uses whatever `flatc` in the `PATH` this means problems.

Reply:
So can't we just make it use `flatc` provided by the subproject and fail otherwise if not found?

Reply:
> So can't we just make it use flatc provided by the subproject and fail otherwise if not found?

Not as per the docs. 

I've pushed a commit where we force the version given by our flatbuffers subproject.